reverse_proxy/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# e.g. github.{{zone}} or cran.{{zone}}
+zone: hic-tre.dundee.ac.uk
```

Comment:
How about change this to `something.example.org` since this is a generic role, and set `zone` in the playbook?

`cran_mirror` below is fine since it's a public mirror that works for everyone.